### PR TITLE
Lesson 3

### PR DESCRIPTION
**Pull request by Hashlemon**
Check it out

### CODE REVIEW

**Comment by romos21** [comment]
@Hashlemon Great ! I see you can do something with setting styles to element and it really GOOD! But I should say something about few moments

File: index.html [hunk 1]
```diff
@@ -180,7 +192,39 @@
...
+            <label><input type="checkbox" name="check" value="3"> Hyper Text and Markup Language</label><br></br>
+            <p>What does HTTP stand for</p>
+            <label><input type="radio" name="radio" value="1"> Hyper Text Translate Protocol</label><br></br>
+            <label><input type="radio" name="radio" value="2"> Hyper Text Transmission Protocol</label><br></br>
+            <label><input type="radio" name="radio" value="3"> Hyper Text Transfer Protocol</label><br></br>
+            <label>Comment:<br></br><textarea name="message" rows="2" cols="30">Comment</textarea></label><br></br>
+            <label>What does HTML stand for?<br></br><select id="select" name="choice">
+                <option value="answer">Your answer</option>
+            </select>
+            </label><br></br>
```
Comment (romos21) [inline index.html:220]: @Hashlemon, I mean close single tag like this <BR/> - not like this <BR></BR> - and generally, as I have already sad, it's not important at all.

**Comment by romos21** [comment]
@Hashlemon , also always try to send correct commit messages whitch describe your actions and code updates - not 'something in the middle' - it's better to add commit like stairs of your homework points

**Comment by Hashlemon** [comment]
> @Hashlemon , also always try to send correct commit messages whitch describe your actions and code updates - not 'something in the middle' - it's better to add commit like stairs of your homework points

Reasonable. But please merge anyway.
Thanx  @romos21

**Comment by romos21** [comment]
@Hashlemon don't see your commit with fixes before merge

**Pull request closed without merging**